plat/kvm/Config.uk

```diff
@@ -45,25 +47,25 @@ config KVM_BOOT_EFI_STUB
 
 endchoice
 
-if KVM_BOOT_EFI_STUB
+if KVM_BOOT_PROTO_EFI_STUB
 
-config KVM_BOOT_EFI_STUB_DEBUG
+config KVM_BOOT_PROTO_EFI_STUB_DEBUG
 	bool "Enable EFI stub crash messages"
 	default n
 
-config KVM_BOOT_EFI_STUB_CMDLINE_FNAME
+config KVM_BOOT_PROTO_EFI_STUB_CMDLINE_FNAME
```

Comment:
Thank you so much for these changes! :pray: My apologies for having missed them.